ballerina/tests/jwt_validator_test.bal

```diff
@@ -17,6 +17,8 @@
 // NOTE: All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
 
 import ballerina/test;
+import ballerina/io;
+import ballerina/crypto;
```

Comment:
Imports are not ordered it seems. We have the imports formatting feature from update9.

Reply:
apply where necessary

Reply:
Fixed with [4171eb3c3fd1372742b8e0f62278ebd1ccdc3287](https://github.com/ballerina-platform/module-ballerina-jwt/pull/1229/commits/4171eb3c3fd1372742b8e0f62278ebd1ccdc3287)